pkg/app/api/grpcapi/web_api.go

```diff
@@ -924,8 +924,11 @@ func (a *WebAPI) GenerateApplicationSealedSecret(ctx context.Context, req *webse
 		return nil, err
 	}
 
-	sse := piped.SealedSecretEncryption
-	if sse == nil {
+	se := piped.SecretEncryption
+	if se == nil {
+		se = piped.SealedSecretEncryption
+	}
+	if se == nil {
```

Comment:
this blown my mind 🤣

Reply:
So do I. `SealedSecretEncryption` is a deprecated field. Seems like code kind of like this appears a couple of time (like for instance pkg/app/piped/cmd/piped/piped.go#441).
I'm gonna say it should be encapsulated by the `config.Piped` because it's an issue itself.

I would propose add the `GetSealedSecretEncryption` method to `config.Piped`
```go
func (p *Piped) GetSealedSecretEncryption() SecretEncryption {
	if p.SecretEncryption != nil {
		return p.SecretEncryption
	}
	return p.SealedSecretEncryption
}
```

Reply:
I get your point, just one thing I wonder 🤔 Should it be `model.Piped` instead of `config.Piped`?

Reply:
To be honest, I was thinking about that: adding that function into both config and model.
But our model is defined by using proto and `Get...` is already generated by protobuf. So I could not find a good name for the function so I decided to do as current.

Reply:
How about `Get...InPriority` ? 🤔

Reply:
that is true. it is `mode.Piped`, not `config.Piped`. Then how about adding a util function like `GetSealedSecretEncryption(piped *Piped)` on model/piped.go? Anyway, what callers have to know the SealedSecretManagement field is deprecated is a bit tedious. We'd better have some kind of things to encapsulate it, right?

Reply:
Sure. Let me add that function.

Reply:
Fixed it!